### PR TITLE
Adding a section about writing log messages

### PR DESCRIPTION
**Pull request by spacekookie**
This adds a section with advice on things to keep in mind when writing logging code (#75)

### CODE REVIEW

**Comment by spacekookie** [comment]
I'm not sure these examples are _super_ good? They do kinda demonstrate things mentioned in the previous section (aka PR #80) though, about showing progress, etc.

File: src/in-depth/human-communication.md [hunk 1]
```diff
@@ -16,18 +16,47 @@ Style of writing:
...
 
-<aside class="todo">
+When communicating non-nominal state it's important to be consistent.
+A heavily logging application that doesn't follow strict logging levels
+provides the same amount or even less information than a non-logging
+application.
 
-**TODO:**
-Log messages:
-- levels
```
Comment (killercup) [inline src/in-depth/human-communication.md]: hm, maybe link to a list of log levels too?

File: src/in-depth/human-communication.md [hunk 2]
```diff
@@ -16,18 +16,60 @@ Style of writing:
...
+provides the same amount, or even less information
+than a non-logging application.
+
+Because of this it's important to define the severity of events
+and messages that are related to it;
+then use consistent log levels for them.
+This way users can select the amount of logging themselves
+via `--verbose` flags.
+
+Available levels should be
```
Comment (killercup) [inline src/in-depth/human-communication.md]: log has trace but not fatal: https://docs.rs/log/0.4.4/log/enum.Level.html (feel free to also link to these docs!)

File: src/in-depth/human-communication.md [hunk 3]
```diff
@@ -16,18 +16,60 @@ Style of writing:
...
+When communicating non-nominal state it's important to be consistent.
+A heavily logging application that doesn't follow strict logging levels
+provides the same amount, or even less information
+than a non-logging application.
+
+Because of this it's important to define the severity of events
+and messages that are related to it;
+then use consistent log levels for them.
+This way users can select the amount of logging themselves
+via `--verbose` flags.
```
Comment (killercup) [inline src/in-depth/human-communication.md]: …or env vars

**Pull request merged**